site/docs/webrtc/webhooks.md

```diff
@@ -18,23 +18,23 @@ import TabItem from "@theme/TabItem";
 
 ### Participant Creation (beta)
 
-The only API call that allows the registration of a callback / webhook is the POST to the participants resource that is used to create a new participant. The callback/webhook URL is declared in the `POST` to the `createParticipants` endpoint. If the `callbackUrl` property is included in the payload, the value associated with that property is assumed to be a URL. That URL (if found to be valid) will be invoked on events that occur as part of the Participant life cycle.
+The only API call that allows the registration of a callback / webhook is the `POST` to the `createParticipants` endpoint that is used to create a new Participant. If the `callbackUrl` property is included in the payload, the value associated with that property is assumed to be a URL which, if found to be valid, will be invoked on events that occur as part of the Participant life cycle.
 
 :::note
 Participant event webhooks are currently in Beta, and must be activated by submitting a support ticket to Bandwidth.
 :::
 
 :::note
-Participant event webhooks may change in the future. All attempts will be made to retain the syntax and semantics of the current implementation, but until Bandwidth completes the process of learning with our customers that the implementation is optimal, we cannot assure it. Bandwidth welcomes any and all comments on the implementation while in the Beta phase.
+Participant event webhooks may change in the future. All attempts will be made to retain the syntax and semantics of the current implementation, but until Bandwidth completes the process of learning with our customers that the implementation is optimal, we cannot guarantee it. Bandwidth welcomes any and all comments on the implementation while in the Beta phase.
 :::
 
 ## Life Cycle Events
 
 The events that will trigger invocation of the specified URL are:
 
-- 'onConnect' - establishment of the websocket connection that will allow the flow of WebRTC Media, an explicit action on the part of the websocket client.
-- 'onDisconnect' - explicit removal of the websocket connection, prompted by explicit server or client action.
-- 'onLeave' - loss of the websocket connection, possibly due to loss of IP connectivity.
+- `onConnect` - an explicit action on the part of the client that establishes a signalling connection to control the flow of WebRTC Media.
+- `onDisconnect` - explicit removal of the signalling connection, prompted by explicit server or client action.
```

Comment:
I feel like the descriptions you have for onDisconnect and onLeave should be reversed (i.e. the onLeave seems to actually describe onDisconnect and vice versa)

Reply:
Something else to think about and consider is what information an application developer needs in order to write their application, and tailoring the content to what an application developer is looking for and needs to know.